4.analyze_data/notebooks/UMAP/scripts/0.UMAP_coordinates.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Generate UMAP coordinates for each plate
+
+# ## Import libraries
+
+# In[1]:
+
+
+import glob
+import pathlib
+import pandas as pd
+import umap
+
+from pycytominer import feature_select
+from pycytominer.cyto_utils import infer_cp_features
+
+
+# ## Set constants
+
+# In[2]:
+
+
+# Set constants
+umap_random_seed = 1234
```

Comment:
Consider changing to 0 as this is the way lab convention.
```suggestion
umap_random_seed = 0
```

Reply:
Changed, I do believe this was started in the CFReT repo so I keep forgetting the change it, thanks for noticing!